call_centre/find.feature

```diff
@@ -48,7 +48,60 @@ Feature: Find an appointment
       | Smith    | ith     | 1                        |
       | Smith    | mit     | 1                        |
 
-  Scenario: Search by email
+  Scenario: Search by full email
+    Given I am on the call centre home screen
+    And there is an appointment for customer with email <email>
+    When I search for an appointment by <email>
+    Then I should see <number of search results>
+  
+  Examples:
+    | email               | number of search results |
+    | j.smith@example.com | 1                        |
+    | j.bloggs@123.com    | 1                        |
+  
+  
   Scenario: Search by partial email
+    Given I am on the call centre home screen
+    And there is an appointment for customer with email <customer>
+    When I search for an appointment by <email>
+    Then I should see <number of search results>
+    
+  Examples:
+    | customer              | email      | number of search results |
+    | j.smith@example.com   | ith@examp  | 1                        |
+    | j.bloggs@123.com      | j.bloggs@  | 1                        |
+  
+  
   Scenario: Search by surname and email
+    Given I am on the call centre home screen
+    And there is an appointment for customer with surname <customer> and email <email>
+    When I search for appointment by <surname> and <email>
```

Comment:
Every `<thingy>` corresponds to a column in the examples table, so if we want to search by the full customer surname:

```gherkin
When I search for appointment by <customer> and <email>
```

or if we are searching by partial surname then:

```gherkin
When I search for appointment by <surname> and <email>

Examples:
   | customer | surname | email | number of search results |
   | Smith    | mit     | ...   | ...                      |

```